todo/urls.py

```diff
@@ -0,0 +1,31 @@
+from django.urls import path
+
+from .views import (
+    TaskListView,
+    TaskCreateView,
+    TaskUpdateView,
+    TaskDeleteView,
+    TagListView,
+    TagCreateView,
+    TagUpdateView,
+    TagDeleteView,
+    TaskChangeStatusView,
+
+)
+
+urlpatterns = [
+    path("", TaskListView.as_view(), name="task-list"),
+    path("task/create", TaskCreateView.as_view(), name="task-create"),
+    path("task/<int:pk>/update", TaskUpdateView.as_view(), name="task-update"),
```

Comment:
The UI of the app should contain the "Undo" and "Complete" button that trigger the requests to this endpoint. However, the buttons are missing and both actions can be done only by following "Update" link